lib/lab_manager/provider/vsphere.rb

```diff
@@ -130,7 +146,7 @@ def instance_uuid
     def vm_data(vm_instance_data = nil, full: false, vs: nil)
       data_proc = lambda do |vs_|
         vm_instance_data ||= vs_.get_virtual_machine(compute.provider_data['id'])
-        vm_instance_data.each_with_object({}) do |s, (k, v)|
+        vm_instance_data.inject({}) do |s, (k, v)|
```

Comment:
whay you put back `inject`. You can use each_with_object and even remove `s` line on the end of the block.